modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -426,6 +426,11 @@ public void bindObjectDefinitions(long[] objectRelationshipIds)
 
 			_objectFieldLocalService.updateRequired(
 				objectRelationship.getObjectFieldId2(), true);
+
+			_updateObjectDefinitionPortlet(
+				objectDefinition1, rootObjectDefinitionId);
```

Comment:
Hi @larissacribeiro this statement should comes before the line 419 because we're handle objectDefinition1 and objectDefinition2 individually,

Thanks!

Reply:
done!